gradle/libs.versions.toml

```diff
@@ -114,5 +112,15 @@ hilt-android-compiler = { group = "com.google.dagger", name = "hilt-android-comp
 hilt-navigationCompose = { group = "androidx.hilt", name = "hilt-navigation-compose", version.ref = "hilt" }
 accompanist-systemUiController = { group = "com.google.accompanist", name = "accompanist-systemuicontroller", version.ref = "accompanist" }
 accompanist-placeholder = { group = "com.google.accompanist", name = "accompanist-placeholder", version.ref = "accompanist" }
-chrisbanes-snapper = { group = "dev.chrisbanes.snapper", name = "snapper", version.ref = "snapper" }
 test-junit = { group = "junit", name = "junit", version.ref = "junit" }
+
+[bundles]
+
+compose = [
+    "compose-animation",
+    "compose-foundation",
+    "compose-material-icons",
```

Comment:
Should we remove this from the bundle since it's not necessarily used if compose is?

Reply:
We can, but is it not necessarily used because we haven't implemented most of the stuff?

Reply:
https://developer.android.com/reference/kotlin/androidx/compose/material/icons/package-summary:
> The most commonly used set of Material icons are provided by androidx.compose.material:material-icons-core - this library is also provided as an API dependency by androidx.compose.material:material.

> A separate library, androidx.compose.material:material-icons-extended, contains the full set of Material icons. Due to the very large size of this library, make sure to use R8/Proguard to strip unused icons if you are including this library as a direct dependency. Alternatively you can make a local copy (by copy and pasting) the icon(s) you wish to keep, or using Android Studio's 'Import vector asset' feature.

I wonder where it's used :thinkies:

Reply:
All I know is random icons aren't available without it lul

Reply:
Removed it.